animation_nodes/nodes/mesh/bmesh_reverse_faces.py

```diff
@@ -0,0 +1,24 @@
+import bpy, bmesh
+from bpy.props import *
+from ... events import executionCodeChanged
+from ... base_types import AnimationNode
+
+class BMeshReverseFacesNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_BMeshReverseFacesNode"
+    bl_label = "BMesh Reverse Faces"
```

Comment:
Change the node name into something more familiar. `BMesh Invert Normals` maybe.